src/callout.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import Stack from './stack';
+import {useTheme} from './hooks';
+import {ThemeVariant, useIsInverseVariant} from './theme-variant-context';
+import {createUseStyles} from './jss';
+import {Text2, Text3} from './text';
+import IconClose from './icons/icon-close';
+import IconButton from './icon-button';
+import classNames from 'classnames';
+import {ButtonLink, ButtonPrimary, ButtonSecondary} from './button';
+import Inline from './inline';
+
+const useStyles = createUseStyles(({colors}) => ({
+    container: {
+        background: colors.backgroundAlternative,
+        borderRadius: 4,
+        padding: 16,
```

Comment:
IIRC this can be done with `Boxed`, also dealing with choosing the outer inverse vs inner inverse

Reply:
no, because Boxed uses different background color logic